rest/src/main/java/org/jboss/pnc/rest/trigger/BuildTriggerer.java

```diff
@@ -144,4 +170,56 @@ private BuildConfigurationAudited getLatestAuditedBuildConfiguration(Integer bui
         }
         return buildConfigRevs.get(0);
     }
+    
+    /*
+     * TODO: Do not ignore certificates, rather setup servers properly.
+     */
+    private void signalBpmEvent(String uri) {
```

Comment:
+1 will be done as part of NCL-866 as we will need it for the BPM integration test.